vars/validateArtifacts.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+SignArtifacts signs the given artifacts and saves the signature in the same directory.
+@param Map[version] <Required> - Version of the opensearch artifact that needs to be validated.
+@param Map[distribution] <Optional> - Choose distribution type among [tar(default),rpm,yum,docker].
+@param Map[architecture] <Optional> - Select the architecture type among [x64(default) and arm64].
+@param Map[platform] <Optional> - The distribution platform.
+@param Map[docker_source] <Optional> - Specify the docker source from [DockerHub(default), ECR] to pull the docker image.
+@param Map[os_build_number]<Optional> - Specify Opensearch build number from opensearchstaging if required.
+@param Map[osd_build_number]<Optional> - Specify Opensearch-Dashboard build number from opensearchstaging if required.
```

Comment:
You can remove this and say pass whatever needs to be passed to validation script. Add more details how they are passed. example `--<key> <value>`

Reply:
Sure @gaiksaya , can I completely remove the above lines and specify in detail how each argument is passed to the workflow with a help statement to understand how to run the shell script?